recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,5 @@
 # https://doc.qt.io/qt-6/macos.html
-MACOSX_DEPLOYMENT_TARGET:  # [osx]
+c_stdlib_version:          # [osx]
   - '11.0'                 # [osx]
```

Comment:
If you use this, you need to add `- {{ stdlib("c") }}` to the build dependencies, otherwise it will not work correctly.

In turn, you can then delete `- __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.14") }}  # [osx and x86_64]`.